classes/form/autocomplete_form.php

```diff
@@ -17,62 +17,80 @@
 /**
  * Implement autocomplete moodle form.
  *
- * @package          tool_wsformat
- * @copyright        2023 Djarran Cotleanu
- * @author           Djarran Cotleanu
- * @license          http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ * @package   tool_wsformat
+ * @copyright 2023 Djarran Cotleanu, Zach Pregl
+ * @author    Djarran Cotleanu, Zach Pregl
+ * @license   http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
  */
 
 namespace tool_wsformat\form;
 
+
 use moodleform;
 
 /**
  * Form for selecting web services to format.
  *
  * @package   tool_wsformat
- * @copyright 2023 Djarran Cotleanu
- * @author    Djarran Cotleanu
+ * @copyright 2023 Djarran Cotleanu, Zach Pregl
+ * @author    Djarran Cotleanu, Zach Pregl
  * @license   http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
  */
 class autocomplete_form extends moodleform {
 
+
     /**
      * Define an autocomplete element for browsing webservices and a submit button.
      */
-    public function definition() {
+    public function definition(): void {
```

Comment:
Moodle coding standards dont need void return to be explicitly declared. The absence of a return type implies void in that case